tests/helper/helper_cmd_wrapper.go

```diff
@@ -117,6 +117,7 @@ func (cw *CmdWrapper) ShouldRun() *CmdWrapper {
 func (cw *CmdWrapper) Should(f func(session *gexec.Session)) {
 	cw.Runner()
 	f(cw.session)
+	cw.session.Wait()
```

Comment:
This seems to fix the problem, logs point so.
```sh
Ran 436 of 788 Specs in 1079.759 seconds
SUCCESS! -- 436 Passed | 0 Failed | 0 Pending | 352 Skipped


Ginkgo ran 1 suite in 19m5.388476487s
Test Suite Passed
```
But I don't understand why this missing piece created a problem in the first place.


I also noticed that if you check this run on IBM cloud, the duration for Job run was `Duration1 hour, 57 minutes, 19 seconds`, but the test run duration was `Duration23 minutes, 40 seconds`, is that normal?

Reply:
> But I don't understand why this missing piece created a problem in the first place.

The tests were terminating before the processes `odo deploy` were terminated. Ginkgo probably has problems handling children processes remaining after the tests finish.

> I also noticed that if you check this run on IBM cloud, the duration for Job run was Duration1 hour, 57 minutes, 19 seconds, but the test run duration was Duration23 minutes, 40 seconds, is that normal?

The job waited 1h34 before to start.